addons/link_tracker/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,18 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_unlink_by_xmlid = [
+    'link_tracker.view_link_tracker_form_stats',
+]
+
+
+def fill_link_tracker_code(env):
+    tracker_codes = env['link.tracker.code'].search([('code', '=', None)])
```

Comment:
```suggestion
    tracker_codes = env['link.tracker.code'].search([('code', '=', False)])
```

Reply:
Isn't None equivalent to Null? why False?